kernel/lib/time.cpp

```diff
@@ -0,0 +1,90 @@
+/**
+ * @file time.hpp
+ * @author Michel (JMallone) Gomes (michels@utfpr.edu.br)
+ * @brief Time lib code 
+ * @version 0.1
+ * @date 2021-07-27
+ *
+ * @copyright Copyright the Panix Contributors (c) 2021
+ *
+ * References:
+ *              https://github.com/sidsingh78/EPOCH-to-time-date-converter/blob/master/epoch_conv.c
```

Comment:
Use 4 spaces instead of 8 here. I think `vim` defaults to a tab length larger than 4, which is what we use for Panix.